### PR TITLE
Add UUID to Mushra results

### PR DESCRIPTION
**Pull request by fzalkow**
In the resulting CSV file, different participants can only be distinguished by their names given in the final questionnaire. This might be error-prone because different participants could use the same name. I added a UUID to avoid this case (only implemented for MUSHRA in the PHP file).

### CODE REVIEW

**Comment by nullpunktTUD** [comment]
Just wanted to chime in and say that this is a feature that would be extremely helpful to me, too.

Also, it looks like the most important change is actually in `Session.js`. The change in the PHP script just reflects the fact that the new data field needs to be written to the results.

Anyway, I would love to see this included for all kinds of pages.

**Comment by fzalkow** [comment]
I agree that the change in the PHP file is trivial.

The main modification is including the UUID package via CDN (unpkg) in `index.html`. I do not know if using CDNs is considered acceptable in this project.

Adding this feature to other pages should be easy. One just needs to push `$session->uuid` to the corresponding results in the PHP file.

File: index.html [hunk 1]
```diff
@@ -42,13 +42,14 @@
...
     <script src ="lib/external/three.js/loaders/ColladaLoader.js"></script>
 
-
     <!--noUiSlider -->
     <script src="lib/external/wmSlider/slider.min.js"></script>
 
     <!-- keyboard support -->
     <script src ="lib/external/mousetrap/mousetrap.min.js"></script>
 
+    <script src="https://unpkg.com/uuid@latest/dist/umd/uuidv4.min.js"></script>
```
Comment (faroit) [inline index.html:51]: can you copy in the dependency and its license?

**Review by faroit — CHANGES_REQUESTED** [review]

**Comment by fzalkow** [comment]
I copied the dependency and the license file. To be completely transparent: The dependency is the minified version of uuidv4 provided by the CDN unpkg. I used the license from the original repo (https://github.com/uuidjs/uuid), which does not contain the minified version. I guess the original license still applies to the minified version... I still want to mention this.

**Review by faroit — APPROVED** [review]
looks good.

**Pull request merged**